packages/editor-ui/src/main.ts

```diff
@@ -44,13 +45,35 @@ if (window.sentry?.dsn) {
 		release,
 		environment,
 		beforeSend(event, { originalException }) {
+			const ignoredErrors = [
```

Comment:
Since this function is called on every error, we will be creating this array each time. This function is (hopefully) not called very often, but just to be on the careful side, can we please move this array outside this function body?

Reply:
Great point! Moved it outside the function.